repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java

```diff
@@ -31,10 +31,10 @@
     private static RepoService repoService;
 
     // update this when new workitem is added
-    private static int TOTAL_MODULES = 40;
+    private static int TOTAL_MODULES = Integer.parseInt(System.getProperty("total.mod.count", "40"));
 
     // update this when new workitem handlers are added
-    private static int TOTAL_SERVICES = 93;
+    private static int TOTAL_SERVICES = Integer.parseInt(System.getProperty("total.wi.count", "93"));
```

Comment:
Here it is 40 for work items and 93 for work item handlers as default, but in the failsafe config it says 6 and 15. Moreover, if we are pretty sure that failsafe will set these properties, why still have hard-coded values?

Reply:
that's the whole point ... there are different values expected when running in full/community mode - meaning all handlers included or with productised only few included. Since the majority of times the values in tests will change when new handlers are added so it's easier to update the test only.

Reply:
I have just noticed that it is only for production ones. Sorry for confusion, forget what I said...